WordPress/Classes/Models/Theme.m

```diff
@@ -30,6 +40,35 @@ + (NSString *)entityName
     return NSStringFromClass([self class]);
 }
 
+#pragma mark - Links
+
+- (NSString *)customizeUrl
+{
+    return [self.blog adminUrlWithPath:ThemeAdminUrlCustomize];
+}
+
+- (NSString *)detailsUrl
+{
+    NSString *homeUrl = self.blog.homeURL.hostname;
+
+    return [NSString stringWithFormat:ThemeUrlDetails, homeUrl, self.themeId];
+}
+
+- (NSString *)demoUrl
```

Comment:
(Sorry i didn't catch this before!). Perhaps this should be a readonly property? (or it should be named in another way). Otherwise the *Getter* would return something different than the stuff stored by the *setter* ?